api/password_handler.py

```diff
@@ -0,0 +1,37 @@
+import json
+from config import MeticulousConfig, CONFIG_SYSTEM, ROOT_PASSWORD
+from .base_handler import BaseHandler
+from .api import API, APIVersion
+from log import MeticulousLogger
+
+logger = MeticulousLogger.getLogger(__name__)
+
+
+class RootPasswordHandler(BaseHandler):
+    def get(self):
+        # Commented for development - Uncomment in production
```

Comment:
You could extend baseHandler and have a "localAccessHandler" which is then used here :)